lib/jsdom/living/nodes/SVGSVGElement.webidl

```diff
@@ -0,0 +1,39 @@
+interface SVGSVGElement : SVGGraphicsElement {
```

Comment:
For SVG especially, we should add links to the specs, since there are so many to choose from.